tools/getopt_long.c

```diff
@@ -0,0 +1,218 @@
+/*
+ * getopt_long() -- long options parser
+ *
+ * Portions Copyright (c) 1987, 1993, 1994
```

Comment:
I guess we should something about this into https://github.com/opencryptoki/opencryptoki/blob/master/COPYRIGHTS.
Same for err.c above.